pldoc/static/js/colors.js

```diff
@@ -0,0 +1,132 @@
+define([
+    'jquery'
+], function($) {
+    'use strict';
+
+    var $pane = $('#colors-preview .info-pane'),
+        $title = $($pane).find('.color-info-title'),
+        $desc = $($pane).find('.color-description'),
+        $back = $($pane).find('.color-combinations.background'),
+        $ref = $($pane).find('.color-reference'),
+        $rgb = $($pane).find('.color-rgb'),
+        $hex = $($pane).find('.color-hex');
+
+    var $button;
+
+    var colors;
+
+    var ColorHandler = {
+
+        init: function() {
+            ColorHandler.setupJSONColors();
+            ColorHandler.listenForClick();
+            ColorHandler.listenForKeypress();
+            ColorHandler.listenForClose();
+        },
+
+        setupJSONColors: function() {
+            $.ajax({
+                async: false,
+                url: '/public/js/colors.json',
+                complete: function(data) {
+                    colors = data.responseJSON;
```

Comment:
Couple things here:

- You're doing an ajax request for a local file, which seems strange, have you looked into just requiring the json with requireJS? I think that's possible, but I could be wrong.
- You don't do anything with `colors` after you get the JSON response?
- Synchronous ajax will lock the browser while this file is fetched, IIRC.